File: src/main/com/stockcomp/StockcompApplication.java
package com.stockcomp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.elasticsearch.repository.config.EnableElasticsearchRepositories;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories(basePackages = "com/stockcomp/repository/jpa")
@EnableElasticsearchRepositories(basePackages = "com/stockcomp/repository/document")
public class StockcompApplication {

    public static void main(String[] args) {
        SpringApplication.run(StockcompApplication.class, args);
    }
}

File: src/main/kotlin/com/stockcomp/StockCompApplication.kt
package com.stockcomp

import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableSchedulerLock(defaultLockAtMostFor = "10m")
class StockCompApplication

fun main(args: Array<String>) {
    runApplication<StockCompApplication>(*args)
}
